tests/metrics/network/network-metrics-iperf3.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# Port number where the server will run
+port=5201:5201
+# Image name
+image=gabyct/network
+# Measurement time (seconds)
+time=5
+
+function setup {
+	runtime_docker
+	kill_processes_before_start
+}
+
+# This script will perform all the measurements using a local setup using iperf3
+
+function iperf3_bandwidth {
+	setup
+	bandwidth_result=$(mktemp)
+	$DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "mount -t ramfs -o size=20M ramfs /tmp && iperf3 -p ${port} -s" > /dev/null && \
```

Comment:
Hi, maybe could be good to write lines with less characters, it could allow a better readability. 80 characters rule sometimes is a little strict, however a shorter lines would be good :), maybe can be something like that:
```
function iperf3_bandwidth(){
        command_string=$(cat << EOF
        mount -t ramfs -o size=20M ramfs /tmp
        iperf3 -p ${port} -s
        EOF)
        $DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "command_string"
}
```